### PR TITLE
Update buttons

### PR DESCRIPTION
**Pull request by khouloudbelguith**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Update buttons
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/21361
| How to test?  | Follow https://github.com/PrestaShop/PrestaShop/issues/21361

### CODE REVIEW

File: ps_mainmenu.php [hunk 1]
```diff
@@ -1116,12 +1116,12 @@ public function renderAddForm()
                             array(
                                 'id' => 'active_on',
                                 'value' => 1,
-                                'label' => $this->trans('Enabled', array(), 'Admin.Global')
+                                'label' => $this->trans('YES', array(), 'Admin.Global')
```
Comment (PierreRambaud) [inline ps_mainmenu.php:1119]:
```suggestion
                                'label' => $this->trans('Yes', array(), 'Admin.Global')
```

File: ps_mainmenu.php [hunk 2]
```diff
@@ -1116,12 +1116,12 @@ public function renderAddForm()
...
                             ),
                             array(
                                 'id' => 'active_off',
                                 'value' => 0,
-                                'label' => $this->trans('Disabled', array(), 'Admin.Global')
+                                'label' => $this->trans('NO', array(), 'Admin.Global')
```
Comment (PierreRambaud) [inline ps_mainmenu.php:1124]:
```suggestion
                                'label' => $this->trans('No', array(), 'Admin.Global')
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by ngodefroy** [comment]
TEST OK

Capture d’écran 2020-10-09 à 13 48 48

Capture d’écran 2020-10-09 à 13 49 01

**Comment by Progi1984** [comment]
Thanks @khouloudbelguith & @ngodefroy

**Pull request merged**